dbrequests/connection.py

```diff
@@ -106,13 +107,26 @@ def _send_data_pandas(self, df, table, pandas_mode='append', **params):
                   index=False, **params)
 
     @contextmanager
-    def _temporary_table(self, table):
+    def _temporary_table(self, table: str, with_cols: (str, None) = None):
```

Comment:
Vermisse noch die doc zu `with_cols`

Reply:
Es gibt in dem Paket keine docstrings für private member Funktionen. with_cols kann auch nicht vom User-gesetzt werden und ist ein Implementierungsdetail. Wir könnten dort einen Kommentar einsetzen, falls es völlig unklar ist, was damit passiert?

Reply:
Für mich ist es auch etwas unklar. Ein Kommentar mit deinem Hinweis @wahani fänd ich gut.

Reply:
Kommentar finde ich gut.

Reply:
Ist eingebaut: https://github.com/INWTlab/dbrequests/pull/39/commits/5915104ae05b13219dea0031560936331a93e909